<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jsj_UserManagerdlg</class>
 <widget class="QWidget" name="jsj_UserManagerdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#jsj_UserManagerdlg{
	border: 3px solid #033b60; /* 边框宽度为1px，颜色为#A0A0A0 */
}
</string>
  </property>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>120</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/jsj/rc/用户管理/返回.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>878</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="AddNewBtn">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>51</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/新增.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="EditBtn">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>51</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/编辑.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DelBtn">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>51</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/删除.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DaoruBtn">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>51</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/导入.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DaochuBtn">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>51</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/导出.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RecordBtn">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>51</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/使用记录.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>230</y>
     <width>270</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="SearchBtn">
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>51</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/检索.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ReSetBtn">
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>51</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/重置.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>810</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/new/jsj/rc/用户管理/IconPass.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="StationEdit">
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
border-image: url(:/new/jsj/rc/用户管理/边框.png);</string>
      </property>
      <property name="placeholderText">
       <string>请输入车站名称</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/jsj/rc/用户管理/IcoUser.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="UserNameEdit">
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;黑体&quot;;
border-image: url(:/new/jsj/rc/用户管理/边框.png);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>请输入用户名名称</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>910</y>
     <width>190</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="UpBtn">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>49</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/上一页.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NextBtn">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/下一页.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>1661</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(18, 81, 116, 180);
font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1590</x>
     <y>160</y>
     <width>151</width>
     <height>65</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>65</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>65</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/jsj/rc/登录界面/维护.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
